src/main/kotlin/ir/ac/ut/jalas/clients/ReservationClient.kt

```diff
@@ -4,8 +4,10 @@ import ir.ac.ut.jalas.clients.models.AvailableRoomsResponse
 import ir.ac.ut.jalas.clients.models.ReservationRequest
 import ir.ac.ut.jalas.clients.models.ReservationResponse
 import org.springframework.cloud.openfeign.FeignClient
+import org.springframework.context.annotation.Profile
 import org.springframework.web.bind.annotation.*
 
+@Profile("!test")
```

Comment:
It it ok that our source code has token effects from test?

Reply:
It's absolutely not OK. I couldn't find any other workarounds to override feign client beans.